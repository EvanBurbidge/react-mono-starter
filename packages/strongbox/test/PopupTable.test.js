import React from 'react';
import {PopupMenu, PopupTable} from '../build/rrpm.module';
import renderer from 'react-test-renderer';

test('PopupTable', () => {
  const component = renderer.create(
    <PopupMenu>
      <PopupTable rowItems={4}>
        <span key={1}>1</span>
        <span key={2}>2</span>
        <span key={3}>3</span>
        <span key={4}>4</span>
      </PopupTable>
    </PopupMenu>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
