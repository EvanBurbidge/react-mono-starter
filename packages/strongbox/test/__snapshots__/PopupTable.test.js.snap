// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopupTable 1`] = `
<div
  className="PopupMenu_PopupMenu__8TfA4"
>
  <div
    className="PopupMenu_button__20m_Y"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="PopupMenu_popover__3hd_Z"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    style={
      Object {
        "height": "auto",
        "left": "calc(-100px + 50%)",
        "visibility": "hidden",
        "width": "calc(200px - 10px)",
      }
    }
  >
    <div
      className="PopupTable_PopupTable__3fGcX"
      style={
        Object {
          "width": "calc(200px - 10px)",
        }
      }
    >
      <div
        className="PopupTable_item__1j75h"
      >
        <span>
          1
        </span>
      </div>
      <div
        className="PopupTable_item__1j75h"
      >
        <span>
          2
        </span>
      </div>
      <div
        className="PopupTable_item__1j75h"
      >
        <span>
          3
        </span>
      </div>
      <div
        className="PopupTable_item__1j75h"
      >
        <span>
          4
        </span>
      </div>
    </div>
  </div>
</div>
`;
